src/shared/util.js

```diff
@@ -11,8 +11,8 @@ const { findIndex } = require('min-dash');
  *
  * @return {Assignee | null}
  */
-function getNextAssignee(candidates, lastAssignee, offset = 1) {
-  const lastIndex = findIndex(candidates, c => c.login === lastAssignee.login);
+function getNextAssignee(candidates, lastAssignee={login:""}, offset = 1) {
```

Comment:
This is not the right fix. Instead we want to handle missing "lastAssignee" gracefully, within our application logic.